internal/scorecard/alpha/testdata/0.0.1/cache.example.com_memcacheds_crd.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Do we want the directory structure for this testdata to match the bundle directory format we expect?
```
testdata
└── bundle
    ├── manifests
    │   ├── cache.example.com_memcacheds_crd.yaml
    │   ├── memcached-operator.v0.0.1.clusterserviceversion.yaml
    │   └── memcached-operator.v0.0.1.clusterserviceversion.yaml.orig
    └── metadata
        └── annotations.yaml
```

Reply:
I updated the testdata to use the new bundle format.  I need to add a correct annotations.yaml file at some point, I was unsure of the format of that, figured it could be done in another PR.

Reply:
For reference about `annotations.yaml`: https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md#docker

Reply:
sweet, just added a sample annotations.yaml file.